play-plugins-mailer do not work properly for other mail port than 465.

I use:

```
    "com.typesafe" %% "play-plugins-util" % "2.2.0",
    "com.typesafe" %% "play-plugins-mailer" % "2.2.0",
```

in my application.conf

smtp {
    host=smtp.live.com
    port=587
    ssl=true
    user="first-creation"
    password=**********
    from="first-creation@hotmail.com"
}

I already set the port to 587 
and then the error for send mail:

```
--- (Running the application from SBT, auto-reloading is enabled) ---

SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/C:/Users/Sheng/.ivy2/cache/ch.qos.logback/log
back-classic/jars/logback-classic-1.0.13.jar!/org/slf4j/impl/StaticLoggerBinder.
class]
SLF4J: Found binding in [jar:file:/C:/Users/Sheng/.ivy2/cache/org.slf4j/slf4j-no
p/jars/slf4j-nop-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.

SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorSta
ticBinder]
[info] play - Listening for HTTP on /0:0:0:0:0:0:0:0:9000

(Server started, use Ctrl+D to stop and go back to the console...)

[info] play - database [default] connected at jdbc:postgresql://localhost:5432/w
emet
[info] application - [securesocial] loaded templates plugin: securesocial.contro
llers.DefaultTemplatesPlugin
[info] play - Starting application default Akka system.
[info] play - Application started (Dev)
[debug] a.c.InMemoryUserService - users = Map()
[ERROR] [04/19/2014 10:03:22.507] [play-akka.actor.default-dispatcher-11] [TaskI
nvocation] Sending the email to the following server failed : smtp.live.com:587
org.apache.commons.mail.EmailException: Sending the email to the following serve
r failed : smtp.live.com:587
        at org.apache.commons.mail.Email.sendMimeMessage(Email.java:1242)
        at org.apache.commons.mail.Email.send(Email.java:1267)
        at com.typesafe.plugin.CommonsMailer.send(MailerPlugin.scala:242)
        at securesocial.core.providers.utils.Mailer$$anonfun$sendEmail$1.apply$m
cV$sp(Mailer.scala:91)
        at akka.actor.Scheduler$$anon$11.run(Scheduler.scala:118)
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(Abst
ractDispatcher.scala:386)
        at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool
.java:1339)
        at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:19
79)
        at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThre
ad.java:107)
Caused by: javax.mail.MessagingException: Could not connect to SMTP host: smtp.l
ive.com, port: 465;  <<<<<<<<<<<<<<<<<<<<<<<why here is still 465?>>>>>>>>>>>>
  nested exception is:
        java.net.ConnectException: Connection timed out: connect
        at com.sun.mail.smtp.SMTPTransport.openServer(SMTPTransport.java:1391)
        at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:41
2)
        at javax.mail.Service.connect(Service.java:310)
        at javax.mail.Service.connect(Service.java:169)
        at javax.mail.Service.connect(Service.java:118)
        at javax.mail.Transport.send0(Transport.java:188)
        at javax.mail.Transport.send(Transport.java:118)
        at org.apache.commons.mail.Email.sendMimeMessage(Email.java:1232)
        ... 10 more
Caused by: java.net.ConnectException: Connection timed out: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method)
        at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketI
mpl.java:79)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.ja
va:339)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocket
Impl.java:200)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java
:182)
        at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:579)
        at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:618)
        at sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:160
)
        at com.sun.mail.util.SocketFetcher.createSocket(SocketFetcher.java:233)
        at com.sun.mail.util.SocketFetcher.getSocket(SocketFetcher.java:163)
        at com.sun.mail.smtp.SMTPTransport.openServer(SMTPTransport.java:1359)
        ... 17 more

[error] a.d.TaskInvocation - Sending the email to the following server failed :
smtp.live.com:587
org.apache.commons.mail.EmailException: Sending the email to the following serve
r failed : smtp.live.com:587
        at org.apache.commons.mail.Email.sendMimeMessage(Email.java:1242) ~[comm
ons-email-1.2.jar:1.2]
        at org.apache.commons.mail.Email.send(Email.java:1267) ~[commons-email-1
.2.jar:1.2]
        at com.typesafe.plugin.CommonsMailer.send(MailerPlugin.scala:242) ~[play
-plugins-mailer_2.10-2.2.0.jar:2.2.0]
        at securesocial.core.providers.utils.Mailer$$anonfun$sendEmail$1.apply$m
cV$sp(Mailer.scala:91) ~[classes/:na]
        at akka.actor.Scheduler$$anon$11.run(Scheduler.scala:118) ~[akka-actor_2
.10-2.2.1.jar:2.2.1]
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-
actor_2.10-2.2.1.jar:2.2.1]
Caused by: javax.mail.MessagingException: Could not connect to SMTP host: smtp.l
ive.com, port: 465
        at com.sun.mail.smtp.SMTPTransport.openServer(SMTPTransport.java:1391) ~
[mail-1.4.1.jar:1.4.1]
        at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:41
2) ~[mail-1.4.1.jar:1.4.1]
        at javax.mail.Service.connect(Service.java:310) ~[mail-1.4.1.jar:1.4.1]
        at javax.mail.Service.connect(Service.java:169) ~[mail-1.4.1.jar:1.4.1]
        at javax.mail.Service.connect(Service.java:118) ~[mail-1.4.1.jar:1.4.1]
        at javax.mail.Transport.send0(Transport.java:188) ~[mail-1.4.1.jar:1.4.1
]
Caused by: java.net.ConnectException: Connection timed out: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.7.0_
55]
        at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketI
mpl.java:79) ~[na:1.7.0_55]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.ja
va:339) ~[na:1.7.0_55]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocket
Impl.java:200) ~[na:1.7.0_55]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java
:182) ~[na:1.7.0_55]
        at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.7.0
_55]
```

--- a/mailer/src/main/scala/com/typesafe/plugin/MailerPlugin.scala
+++ b/mailer/src/main/scala/com/typesafe/plugin/MailerPlugin.scala
@@ -242,6 +242,9 @@ class CommonsMailer(smtpHost: String,smtpPort: Int,smtpSsl: Boolean, smtpTls: Bo
     email.setHostName(smtpHost)
     email.setSmtpPort(smtpPort)
     email.setSSLOnConnect(smtpSsl)
+    if (smtpSsl) {
+      email.setSslSmtpPort(smtpPort.toString());
+    }
     email.setStartTLSEnabled(smtpTls)
     for(u <- smtpUser; p <- smtpPass) yield email.setAuthenticator(new DefaultAuthenticator(u, p))
     email.setDebug(false)

